com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumeVoxelization.compute

```diff
@@ -71,24 +71,20 @@ float ComputeFadeFactor(float3 coordNDC, float dist,
     float  dstF = Remap10(dist, rcpDistFadeLen, endTimesRcpDistFadeLen);
     float  fade = posF.x * posF.y * posF.z * negF.x * negF.y * negF.z;
 
-    return dstF * (invertFade ? (1 - fade) : fade);
-}
-
-float SampleVolumeMask(DensityVolumeEngineData volumeData, float3 voxelCenterNDC, float3 duvw_dx, float3 duvw_dy, float3 duvw_dz)
-{
-    // Scale and bias the UVWs and then take fractional part, will be in [0,1] range.
-    float3 voxelCenterUVW = frac(voxelCenterNDC * volumeData.textureTiling + volumeData.textureScroll);
+    // We only apply exponential falloff on the Blend Distance and not Distance Fade
+    if (falloffMode == DENSITYVOLUMEFALLOFFMODE_EXPONENTIAL)
+        fade = pow(fade, 2.2);
```

Comment:
Exponential is e^x, this is just a power (gamma) function.
Also: remove magic number.